kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -16,7 +16,7 @@ pushd $appliance_root
 popd
 
 mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $vmdb_root
+git clone --separate-git-dir /tmp/manageiq.git <%= @manageiq_checkout.remote %> $vmdb_root
```

Comment:
Keeping the git dir will full history was intentional as it allows the appliance to be used by developers as well.  If we decide to go this route, then we would need some new instructions for developers, and additionally, we should be consistent with the other git repos that we clone.